nrf_modem/doc/sockets/socket_options_func.rst

```diff
@@ -111,10 +111,10 @@ NRF_SO_REUSEADDR
   The default value for :c:macro:`NRF_SO_REUSEADDR` is ``off``, that is, reuse of local addresses is not permitted.
 
 NRF_SO_RCVTIMEO
-   Set a timeout value for the :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` operations.
+   Set a timeout value for the :c:func:`nrf_recv`, :c:func:`nrf_recvfrom` and :c:func:`nrf_accept` operations.
    This option accepts an :c:struct:`nrf_timeval` structure with a number of seconds and microseconds specifying the limit on how long to wait for an input operation to complete.
-   If a receive operation has blocked for this much time without receiving additional data, it returns with a partial count, or ``errno`` is set to ``NRF_EAGAIN`` or ``NRF_EWOULDBLOCK`` if no data were received.
-   The default for this option is the value ``0``, which indicates that a receive operation will not time out.
+   If a receive or accept operation has blocked for this much time without receiving additional data, it returns with a partial count, or ``errno`` is set to ``NRF_EAGAIN`` or ``NRF_EWOULDBLOCK`` if no data were received.
```

Comment:
Will `nrf_accept()` ever return with a "partial count"? The `nrf_accept()` seems to return with either `-1` or file descriptor.

I see this sentence originates from [here](https://pubs.opengroup.org/onlinepubs/9699919799/functions/V2_chap02.html#tag_15_10_16:~:text=If%20a%20receive%20operation%20has%20blocked%20for%20this%20much%20time%20without%20receiving%20additional%20data%2C%20it%20shall%20return%20with%20a%20partial%20count%20or%20errno%20shall%20be%20set%20to%20%5BEAGAIN%5D%20or%20%5BEWOULDBLOCK%5D%20if%20no%20data%20were%20received.).

Reply:
Good point, maybe it's best to split the sentence in two, one for receive operations and another for accept operations?

Reply:
Will split it in two sentences.